README.md

```diff
@@ -418,37 +480,23 @@ This module is used to load features from feature store into your models and scr
 
 Two public classes are exposed form this module. **DatabricksLoader**(DataLoader), **PysparkFeatureLoader**(FeatureLoaderInterface).
 
-### DatabricksLoader
-This is a support class for feature loader and provides the data reading capability from the feature store.
-
-This class needs to be instantiated with an active spark session and a path to the feature store schema (in the format of "catalog_name.schema_name").
-Optionally a date_column information can be passed, otherwise it defaults to use INFORMATION_DATE
-```python
-from rialto.loader import DatabricksLoader
-
-data_loader = DatabricksLoader(spark= spark_instance, schema= "catalog.schema", date_column= "INFORMATION_DATE")
-```
-
-This class provides one method, read_group(...), which returns a whole feature group for selected date. This is mostly used inside feature loader.
-
 ### PysparkFeatureLoader
 
 This class needs to be instantiated with an active spark session, data loader and a path to the metadata schema (in the format of "catalog_name.schema_name").
 
 ```python
 from rialto.loader import PysparkFeatureLoader
 
-feature_loader = PysparkFeatureLoader(spark= spark_instance, data_loader= data_loader_instance, metadata_schema= "catalog.schema")
+feature_loader = PysparkFeatureLoader(spark= spark_instance, feature_schema="catalog.schema", metadata_schema= "catalog.schema2", date_column="information_date")
```

Comment:
looks like you removed the databricks reference explicitly to something more generic, but you carrying the concept of catalog, which is databricks specific

Reply:
obviously this carries through to the rest of the changes too

ie you reference pysparkfeature loader and not toegther with DatabricksLoader anymore